src/components/maintenance/maintenance-requests-tab.js

```diff
@@ -30,64 +37,123 @@ const getSanitizedQueryFromRequest = request => {
 
   const pageNumber = Number(query.page);
   const pageSizeNumber = Number(query.pageSize);
+  const registeredFromMilliseconds = parseInt((query.registeredFrom || '').trim(), 10);
+  const registeredFrom = !isNaN(registeredFromMilliseconds) ? new Date(registeredFromMilliseconds) : null;
+  const registeredUntilMilliseconds = parseInt((query.registeredUntil || '').trim(), 10);
+  const registeredUntil = !isNaN(registeredUntilMilliseconds) ? new Date(registeredUntilMilliseconds) : null;
+  const daysOfWeek = query.daysOfWeek ? query.daysOfWeek.trim().split(',').map(text => Number(text)) : Object.values(DAY_OF_WEEK);
 
   return {
+    filter: (query.filter || '').trim(),
+    registeredFrom,
+    registeredUntil,
+    daysOfWeek,
     page: !isNaN(pageNumber) ? pageNumber : 1,
     pageSize: !isNaN(pageSizeNumber) ? pageSizeNumber : 10
   };
 };
 
-function MaintenanceRequestsTab({ fetchingData, documentsWithRequestCounters }) {
+function MaintenanceRequestsTab() {
```

Comment:
It's really good to see that the tabs on the maintenance page get independent. Maybe we'll have some opportunity to also do this for the other tabs. :+1: